src/Cryptol/TypeCheck/Solver/Class.hs

```diff
@@ -348,5 +436,97 @@ solveLiteralInst val ty
       TVar _ -> Unsolved
 
       _ -> Unsolvable $ TCErrorMessage $ show
-         $ "Type" <+> quotes (pp ty) <+> "does not support literals."
+         $ "Type" <+> quotes (pp ty) <+> "does not support integer literals."
+
+
+-- | Add propositions that are implied by the given one.
```

Comment:
We don't actually use this function right? I think we can remove it.

Reply:
Hm, I thought that might be part of your super-classing stuff, but maybe I messed something up while I was merging?  If you don't recognize it we should probably get rid of it.

Reply:
I removed it at some point because it's unused.  Probably it got restored when you did a rebase.

Reply:
OK, I deleted it again